src/darwin/Framework/CHIP/MTRDeviceClusterData.h

```diff
@@ -26,6 +26,7 @@ NS_ASSUME_NONNULL_BEGIN
 MTR_TESTABLE
 @interface MTRDeviceClusterData : NSObject <NSSecureCoding, NSCopying>
 @property (nonatomic, nullable) NSNumber * dataVersion;
+@property (nonatomic, nullable) NSNumber * pendingDataVersion; // for holding new data version during a report
```

Comment:
Should document here, or in the impl, or both, that this is not encoded/decoded?

Reply:
Added light documentation in comment.